x/gov/module.go

```diff
@@ -182,9 +182,9 @@ type ModuleOutputs struct {
 
 func ProvideModule(in ModuleInputs) ModuleOutputs {
 	defaultConfig := govtypes.DefaultConfig()
-	if in.Config.MaxMetadataLen != 0 {
-		defaultConfig.MaxMetadataLen = in.Config.MaxMetadataLen
-	}
+	defaultConfig.MaxTitleLen = max(in.Config.MaxTitleLen, defaultConfig.MaxTitleLen)
```

Comment:
We shouldn't take the max. If a chain wants it is to be lower, it should be allowed.
The user-defined configuration should take precedence over the SDK defaults.

Reply:
Good point, the AI was trolling me in the comment